### ISSUE TITLE
Topic Text Width: Document Property for Max-Width

### ISSUE THREAD

**Issue by abhineet255** [xmindltd/xmind#339]
My topics tend to be verbose which makes the topics very wide. The map is highly unreadable. There should be a property on the document/map to specify maximum width of each topic. Text longer than the width should automatically wrap.

I can still edit each topic to reduce its width, but it is highly unproductive to do that for each topic individually.

This feature request was raised years ago: https://github.com/xmindltd/xmind/issues/108. But was closed without action.

**Comment by madam-anal** [comment]
Yes, this is a much needed feature. If it gets too tiresome to manually resize each topic you can use FreeMind, which has a default limit.
